### DISCUSSION TITLE
Switch to last message in Kafka topic on start consuming

### DISCUSSION THREAD

**Discussion by dimbler** [ag2ai/faststream#1502] [Q&A]
Dear Team,
Please clarify how can it possible to [seek to end](https://aiokafka.readthedocs.io/en/stable/api.html#aiokafka.AIOKafkaConsumer.seek_to_end) of the Kafka topic partition on subscriber startup?

**Comment by Lancetnik** [comment]
If you are using raw FastStream subscriber, it uses `auto_offset_reset="latest"` already and should read the messages since the moment it started

**Reply by dimbler** [reply]
Dear [Lancetnik](https://github.com/Lancetnik),
Thank you for the rapid answer.
If you metioned about first start, it has been worked, but we need read only last message in topic on every application start. Regardless of whether new messages have appeared or not.

**Reply by Lancetnik** [reply]
Do you want to just `getone` latest message and doesn't consume whole stream?

**Reply by Lancetnik** [reply]
You can make it manually by

```python
subscriber = broker.subscriber("test-topic")

@subscriber  # regular handler
async def handler(): ...

@app.after_startup
async def get_last_msg():
    await subscriber.get_msg()
    # or by access to raw consumer
    await subscriber.consumer.getone()
```
